### ISSUE TITLE
[Feature Request/Question] Ability to play two or multiple Audio Tracks at the same time

### ISSUE THREAD

**Issue by DiaBomb** [clsid2/mpc-hc#1040]
The screen recording software Shadowplay by NVIDIA has a feature that splits the desktop audio and microphone audio into two tracks in a video file. This is very useful for Audio mixing in Video editing. However, as I have a lot of randomly recorded footage, it would be nice to have a quick solution to view them with both audio tracks playing, without having to start a video editor and make a new project just to view the video.

Tldr: I can use this feature for quick access to video with two audio tracks.

If this is already possible then call me blind, because I couldn't find anything online. Only for VLC, but even that is half broken.

**Comment by clsid2** [comment]
Not possible if both tracks are in one file.

If you extract one track into a separate audio file and name it same as the video, then MPC-HC can load it automatically as an audio dub. You also need to disable the internal streams switcher to play both streams simultaneously.

Any other way is not available, and will also not be added.
